src/sentry/static/sentry/app/views/issueList/createSavedSearchButton.tsx

```diff
@@ -95,12 +104,11 @@ class CreateSavedSearchButton extends React.Component {
     return (
       <Access organization={organization} access={['org:write']}>
         <Button
-          title={withTooltip ? t('Add to organization saved searches') : null}
+          title={withTooltip ? t('Add to organization saved searches') : undefined}
           onClick={this.onToggle}
           data-test-id="save-current-search"
           size="zero"
           borderless
-          containerDisplayMode="inline-flex"
```

Comment:
not sure if this is used anymore but does not exist on `<Button>` props

Reply:
This was for the tooltip component, not sure if it's still used